engine/src/main/scala/cromwell/services/KeyValueServiceActor.scala

```diff
@@ -0,0 +1,72 @@
+package cromwell.services
+
+import akka.actor.{Actor, ActorRef, Props}
+import com.typesafe.config.Config
+import cromwell.backend.BackendJobDescriptor
+import cromwell.engine.db.DataAccess
+import cromwell.services.KeyValueServiceActor._
+import cromwell.services.ServiceRegistryActor.ServiceRegistryMessage
+
+import scala.concurrent.Future
+import scala.util.{Failure, Success}
+
+object KeyValueServiceActor {
+  trait KvMessage
+  trait KvAction extends KvMessage with ServiceRegistryMessage {
+    def serviceName = "KeyValue"
+  }
+  trait KvResponse extends KvMessage
+
+  case class KvPut(jobDescriptor: BackendJobDescriptor, key: String, value: Option[String]) extends KvAction
+  case class KvGet(jobDescriptor: BackendJobDescriptor, key: String) extends KvAction
+
+  case class KvPair(jobDescriptor: BackendJobDescriptor, key: String, value: Option[String]) extends KvResponse
```

Comment:
I don't know, I'm dragging my feet because I *really* don't agree but I feel if I try to make my case I'll risk feeling even more alienated (considering it's now 3 against 1), so as long as I don't have to take ownership of this code I'll feel *much* better about these changes...

Just so you know though... we're now going to have this interface for constructing `KvPut`s and `KvGet`s:

```scala
KvPut(KvPair(ScopedKey(callBKey, "k"), Option("value_b")))
KvGet(ScopedKey(callAKey, "k"))
```